src/refda/amm/tbr.h

```diff
@@ -65,6 +65,10 @@ typedef struct
      */
     bool enabled;
 
+    /** Current number of executions
```

Comment:
The AMM is unclear about this, but this (maybe) should be clarified as the number since the rule was enabled. Thoughts?

Reply:
Count since rule was enabled seems to make more sense than an absolute count. I've updated the implementation accordingly.